openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -531,6 +531,31 @@ def from_published_scene(self, replace_in_path=True):
         return replace_with_published_scene_path(
             self._instance, replace_in_path=replace_in_path)
 
+    def get_job_info_through_camera(self, camera=None):
+        """Get the job parameters for deadline submission when
+        multi-camera is enabled.
+        Args:
+            infos(dict): a dictionary with job info.
+        """
+        pass
+
+    def get_plugin_info_through_camera(self, camera=None):
+        """Get the plugin parameters for deadline submission when
+        multi-camera is enabled.
+        Args:
+            infos(dict): a dictionary with plugin info.
+        """
+        pass
+
+    def _use_published_name_for_multiples(self, data):
+        """Process the parameters submission for deadline when
+            user enables multi-cameras option.
+        Args:
+            job_info_list (list): A list of multiple job infos
+            plugin_info_list (list): A list of multiple plugin infos
+        """
+        pass
+
```

Comment:
I actually put this for just in case if there is any other host using it and we can make adjustment on top of it. But yeah, maybe I just remove all these first and we can implement it later if there are other hosts using that.